samples/rest/BingSpellCheckV7.py

```diff
@@ -33,13 +41,13 @@
 
 # Call the API
 try:
-    response = requests.get(endpoint, headers=headers, params=params)
+    response = requests.post(endpoint, headers=headers, params=params)
     response.raise_for_status()
 
-    print("\nHeaders:\n")
+    print('\nHeaders:\n')
     print(response.headers)
 
-    print("\nJSON Response:\n")
+    print('\nJSON Response:\n')
     pprint(response.json())
 except Exception as ex:
-    raise ex
\ No newline at end of file
+    raise ex
```

Comment:
Check that link: https://stackoverflow.com/a/40280880/6941952
TL; DR You caught the exception to raise it as it is. You can do more by changing/customizing the error message, otherwise, the exception handling here is meant not to handle the exception.

Reply:
This is left unchanged from earlier commits, will add "better exception handling" to the backlog, thanks